src/core/ext/filters/client_channel/subchannel_index.cc

```diff
@@ -39,38 +39,37 @@ static gpr_mu g_mu;
 static gpr_refcount g_refcount;
 
 struct grpc_subchannel_key {
```

Comment:
While we're at it, would it also make sense to also replace `grpc_subchannel_key` with `grpc_channel_args`?

Feel free to leave this for a separate PR if you'd prefer.

Reply:
Maybe. I will put together a PR to see whether that's better.